src/main/java/Duke.java

```diff
@@ -4,23 +4,23 @@ public class Duke {
     public static void main(String[] args) {
         Scanner inputScanner = new Scanner(System.in);
         String userInput;
-        String[] userInputArray;
+        String[] userInputs;
 
         // Greeting the user
         Messages.hello();
         
         do {
             userInput = inputScanner.nextLine();
-            userInputArray = userInput.split(" ", 0);
-            switch(userInputArray[0]) {
+            userInputs = userInput.split(" ", 0);
+            switch(userInputs[0]) {
```

Comment:
Consider naming userInputs[0] as a more relatable variable

```suggestion
String command = userInputs[0];
            switch(command) {
```